bin/ert

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```

Comment:
Any reason to move this one out of the Python module? We have had good experience lately with keeping the `bin`-folder (if necessary to exist at all) within the module itself and use entry points to make binaries when being installed.

Reply:
Yes, this is part of a strategy to get a working binary with and without `setuptools`, i.e. use `entry_points` as the entry point for the installed package, `bin/ert` for when you want to run e.g. the checked out code.

But me add those missing bits so this makes sense.

This also is consistent with how e.g nose, bandit, pyflakes, pylint are distributed.

Reply:
Never mind, let's defer this, it is not critical right now.

Reply:
Alright 👍 I’m happy to have a discussion regarding this though. Whatever we find best should probably be implemented in spinningjenny and Everest also...

Reply:
See https://github.com/equinor/ert/issues/418